lib/ansible/module_utils/network/vyos/argspec/static_routes/static_routes.py

```diff
@@ -0,0 +1,84 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+"""
+The arg spec for the vyos_static_routes module
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+class Static_routesArgs(object):  # pylint: disable=R0903
+    """The arg spec for the vyos_static_routes module
+    """
+    def __init__(self, **kwargs):
+        pass
+
+    argument_spec = {
+        'config': {
+            'elements': 'dict',
+            'options': {
+                'address': {
+                    'required': True,
+                    'type': 'str'
+                },
+                'blackhole_config': {
+                    'options': {
+                        'type': {
+                            'type': 'str'
+                        },
+                        'distance': {
+                            'type': 'int'
+                        }
+                    },
+                    'type': 'dict'
+                },
+                'next_hop': {
+                    'elements': 'dict',
+                    'options': {
+                        'address': {
```

Comment:
I believe this is the next-hop address? If so, please update the name to `forward_router_address` in order to match the model with others.